app/views/tasks/index.html.erb

```diff
@@ -0,0 +1,56 @@
+<h1>TASKS</h1>
+
+<div class="grid-wrapper">
+  <div class="left-column">
+    <div class="left-buttons">
+      <%= link_to "Create New Task", new_task_path, method: :get %>
+    </div>
+    <div class="left-buttons">
+      <%= link_to "All Tasks", tasks_path %>
+    </div>
+  </div>
+
+  <div class="right-column">
+    <% if @tasks.length > 0 %>
+      <ul>
+        <% @tasks.each do |task| %>
+          <li>
+            <% if task.completed_at.nil? %>
+              <hr><%= link_to task.name, task_path(task.id) %><%=" - " + task.description %>
+              <br />
+              <div class="task-buttons">
+                <%= button_to "🔲 Mark Complete?", complete_task_path(task.id), method: :patch %>
+              </div>
+            <% else %>
+              <hr><s><%= link_to task.name, task_path(task.id) %><%=" - " + task.description %></s>
+              <br />
+              <div class="task-buttons">
+                <%= button_to "✅ Unmark Complete?", complete_task_path(task.id), method: :patch %>
+              </div>
+            <% end %>
+
+            <div class="task-buttons">
+              <%= button_to "Edit", edit_task_path(task.id), method: :get %>
+            </div>
+            <div class="task-buttons">
+              <%= button_to "Delete", task_path(task.id), method: :delete, data: {confirm: "Are you sure?"}%>
+            </div>
+            <!--      new if statement so that this will print on a new line under the flex box of buttons-->
+            <% if !task.completed_at.nil? %>
+              <br />
+              <%= "Completed: " + task.completed_at %>
+            <% end %>
+          </li>
+        <% end %>
+      </ul>
+    <% else %>
+      <h3>No tasks in the system yet😥 time to add some tasks!</h3>
+    <% end %>
```

Comment:
This conditional is sooo coooool! I was really happy when I saw there was a message for no tasks. Great touch, I think this is awesome. Keep thinking up good ideas like this!